exist-core/src/test/xquery/xquery3/serialize.xql

```diff
@@ -265,7 +265,7 @@ function ser:adaptive-simple-atomic($atomic as xs:anyAtomicType) {
 
 declare
     %test:args(1234)
-    %test:assertEquals('1234')
+    %test:assertEquals(1234)
```

Comment:
Seems to be unrelated to tumbling window

Reply:
It is simply reseting something that has been incorrectly changed since I forked the code to work on Window Expressions.